lib/parsers/log-parser.js

```diff
@@ -76,10 +85,27 @@ export default class LogParser extends Parser {
 
       match = line.match(ERROR_PATTERN)
       if (match) {
+        let outMessageText
         const lineNumber = match[2] ? parseInt(match[2], 10) : undefined
+        // Check if the error is "Undefined control sequence"
+        let errMessageText = match[4]
+        if (errMessageText === UNDEF_CONTROL_SEQ_TEXT) {
+          // The actual macro is the last thing
+          // in the sentence of the next line
+          let nextLine = lines[index + 1]
```

Comment:
I guess in most situations this will be safe given how the error is logged, but if we wanted to be super safe we should probably check that this index lookup is not out of bounds. One possibility is to extend the condition above like

``` js
if (errMessageText === UNDEF_CONTROL_SEQ_TEXT && lines.length > index) {
```

Although that might be wrong place to short-circuit this depending upon what error message you want to give to the user in that case. If you're pretty sure we'll never go out of bounds here, we can skip this check.

Reply:
I can't think of a scenario where the the error would be `Undefined control sequence` and no control sequence will follow in the line below. However, I'll do an out-of-bounds check, to be sure?